packages/redux/src/forms/actions/factories/fetchFormSchemaFactory.ts

```diff
@@ -52,7 +55,7 @@ export default getFormSchema =>
       });
 
       return result;
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
You have Error type in global types in the root of client package.

Reply:
Thanks! However, as Javascript does not support multiple catch clauses, we cannot use `Error` here (which also could be misleading typescript because the actual error thrown might not be an `Error` but something else that was thrown somewhere in code that is not ours). This raises a good question because we are not checking if the error is actually an Error and our error reducers are typed as returning an Error always. We should discuss it further internally but right now, let's leave the `any`. What do you think?

Reply:
At least I think we should do what others are doing typing this error, maintaining the consistency between areas (except if this action is different than others). If you think the global approach it's not ok, an issue should be raised or we should talk about this in some sync.

Reply:
Agreed. However, when we migrate to typescript 4.0 we will have an error if the catch variable is not typed as any because it would be typed as `unknown` which is correct. I will bring that topic in a sync for us to discuss. In the meanwhile, @danielbento92, can you revert this to have the error variable not being explicitly typed as any?